src/conventions.jl

```diff
@@ -2,39 +2,6 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
-# --------------------
-# ORDERING CONVENTION
-# --------------------
-
-"""
-    OrderingConvention
-
-Convention used to reconstruct the faces of a polytope
-from a list of vertices.
-"""
-abstract type OrderingConvention end
-
-"""
-    connectivities(polytope, rank, convention)
-
-Return the list of `Connectivity` elements used to construct all
-faces of rank `rank` of a polytope, following an ordering `convention`.
-"""
-function connectivities(polytope::Type{<:Polytope}, rank::Integer,
-                        convention::Type{<:OrderingConvention})
-  connectivities(polytope, Val(rank), convention)
-end
-
-# ----------------
-# IMPLEMENTATIONS
-# ----------------
-
-include("conventions/GMSH.jl")
-
-# ---------------------
-# ROTATION CONVENTIONS
-# ---------------------
-
```

Comment:
Why was it deleted? Also, there are a few places where `GMSH` is still referred to, if you really want to remove that. However I believe this is unrelated to topological structures; these conventions are just a way to build predefined sets of faces from predefined polytopes.

Reply:
A solution to keep the old functionality would be to change these `connectivities` into constructors for an `ExplicitStructure` instead.

Reply:
The GMSH convention was really buggy in the sense that it was using local indices as opposed to global indices in a vector of points. So getting the connectivities with local indices wasn't very useful. We can build that more generally with these data structures. So I think I will erase all occurrences of GMSH and we can see if it makes sense to bring it back in the future. Right now it was just hanging there with no particular good use case.